.github/workflows/build.yml

```
@@ -63,30 +63,69 @@ jobs:
   release:
     if: github.ref_type == 'tag'
     needs:
-    - build
+    - integration-test
     runs-on: ubuntu-24.04
     permissions:
       contents: write
+      id-token: write
+      attestations: write
+    strategy:
+      max-parallel: 1
+      matrix:
+        os:
+        - linux
+        - darwin
+        - windows
+        arch:
+        - amd64
+        - arm64
 
     steps:
-    - id: generate-version
-      name: Generate Version
+    - name: Checkout
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+
+    - name: Setup
+      uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5
+      with:
+        go-version-file: go.mod
+
+    - name: Generate Version
       env:
-        ref_name: "${{ github.ref_name }}"
-      run: |
-        echo "::set-output name=version::${ref_name#v}"
+        REF_NAME: ${{ github.ref_name }}
+      run: echo "VERSION=${REF_NAME#v}" >> $GITHUB_ENV
```

actionlint suggests using
```suggestion
      run: echo "VERSION=${REF_NAME#v}" >> "$GITHUB_ENV"
```